dex-it-common/src/main/scala/com/wavesplatform/dex/it/config/Implicits.scala

```diff
@@ -0,0 +1,20 @@
+package com.wavesplatform.dex.it.config
+
+import com.typesafe.config.{Config, ConfigRenderOptions}
+
+object Implicits {
+  final implicit class ConfigOps(val self: Config) extends AnyVal {
+    def rendered: String =
+      self
+        .resolve()
+        .root()
+        .render(
+          ConfigRenderOptions
+            .concise()
+            .setOriginComments(false)
+            .setComments(false)
+            .setFormatted(true)
+            .setJson(false)
+        )
+  }
+}
```

Comment:
Move to _com.wavesplatform.dex.settings.utils.ConfigOps_

Reply:
I can't, it used in `DexContainer` and `WavesNodeContainer`, those are in `dex-it-common`

Reply:
`dex-it-common` depends on `dex`, so it should be possible

Reply:
Done